src/actions/mole-game-action-creator.js

```diff
@@ -0,0 +1,21 @@
+const BaseActionCreator = require('./base-action-creator');
+
+export default class MoleGameActionCreator extends BaseActionCreator {
+  // Action types
+  static ACTIVATE_PANEL = "activate-panel";
+  static DEACTIVATE_PANEL = "deactivate-panel";
+
+  /**
+   * Signals the mole game to activate a panel, typically from a timer
+   */
+  sendActivatePanel({stripId, panelId}) {
```

Comment:
It doesn't press the panel, it activates it (i.e. makes is touchable in the mole game).

We should probably do a sprint on terminology where we define "active" as well as panel vs. tile vs. strip.